### ISSUE TITLE
Use fig.alt in the vignette

### ISSUE THREAD

**Issue by MichaelChirico** [kevinushey/rex#65]
This figure doesn't have alt text, reducing its accessibility:

https://github.com/kevinushey/rex/blob/7d174c0d2045162aae7ffc4be75e244a22add73a/vignettes/log_parsing.Rmd#L69-L72

With the new `fig.alt` option in `knitr`, we could fix this:

https://github.com/yihui/knitr/commit/17a64ff10e9f97697178bf79de279aa668def830
